.coveragerc

```diff
@@ -0,0 +1,12 @@
+[run]
```

Comment:
1. Yes, `.coveragerc` is the configuration for running `coverage.py`. More details [here](https://coverage.readthedocs.io/en/coverage-4.3.1/config.html). In our case I am using it mostly to:
   - enable branch covering, which is useful to see whether tests cover multiple options in `if.. elif.. else` blocks
   - allow for collecting results inside tox virtualenvs
2. Yes, coverage is enabled in tox too. This is configured in the `tox.ini`. I've chosen to enable coverage only on python3.4 for now. In the future we can report coverage on all python versions and then combine them all, but [this will require additional work](https://hynek.me/articles/testing-packaging/)